pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -144,11 +240,70 @@ def sign(config) {
     }
 }
 
+def listArchives() {
+    return sh(
+            script: """find workspace/target/ | egrep '.tar.gz|.zip'""",
+            returnStdout: true,
+            returnStatus: false
+    )
+            .trim()
+            .split('\n')
+}
+
+def writeMetadata(config, filesCreated) {
```

Comment:
I'm guessing we could split out all of our metadata functionality into its own script?

Reply:
having separate scripts in jenkins is a pain, if you can manage it let me know